packages/itwinui-css/src/utils/icon/icon-component.scss

```diff
@@ -18,7 +33,7 @@
 
 /// @arg $size - must be one of: 's', 'm', 'l', '2xl', '3xl'
 @mixin svg-icon-size($size) {
-  --_iui-svg-size: #{map-get($iui-icon-sizes, $size)};
+  --_iui-svg-size: #{map-get($icon-sizes, $size)};
```

Comment:
If you are using `map.get` on [L21](https://github.com/iTwin/iTwinUI/pull/854/commits/1a3fb7eeec869b4e906dd1e4375be175a6fef823#diff-b0734b3e90e13cc37780569bf40d58ef79c786c43f23240f1ae7ad180edf9b0bR21), this could be `map.get` too

Reply:
updated